docs/management.md

```diff
@@ -48,16 +48,22 @@ This guide covers:
  * How to [enable HTTPS for management UI](#single-listener-https) and its underlying API
  * How this plugin [operates in multi-node clusters](#clustering)
  * How to [disable metric collection](#disable-stats) to use [Prometheus](./prometheus) exclusively for monitoring
- * [OAuth 2](#oauth2-authentication) support
+ * [OAuth 2](#oauth2-authentication) support:
+    * [Basic Configuration](#basic-configuration)
+    * [Configure client secret](#configure-client-secret)
```

Comment:
What is "Basic Configuration"? It makes me think there is another type of configuration - Advanced maybe, can we use a better title here? Is Configure client secret and the subsequent sections after basic configuration related/part of the "Basic configuration" or are they something else I think we need very clear logical titles here as this TOC at the top of the page is very useful to our users but only if it has a logical flow.

Reply:
I have changed the title to "Minimum .. " rather than "Basic .. "..  The other entries are relevant depending on the use case .. and therefore they are optional depending on the use case. For instance, those users interested in "Identity-Provider initiated logins" must apply the configuration explained in that section. Whereas others, would imply ignore that section...